### PR TITLE
samples: wifi: Scan sample documentation

### PR DESCRIPTION
**Pull request by D-Triveni**
Documentation for Wi-Fi scan sample

Signed-off-by: Triveni D <triveni.danda@nordicsemi.no>

### CODE REVIEW

**Review by richabp — CHANGES_REQUESTED** [review]
I am unable to add my comments after line 10, not sure why. 
Don't we have any Dependencies for this sample, if yes, then please add them.

**Comment by D-Triveni** [comment]
> I am unable to add my comments after line 10, not sure why. Don't we have any Dependencies for this sample, if yes, then please add them.

@richabp Sample has no dependencies.

**Review by richabp — CHANGES_REQUESTED** [review]
Not able to provide my suggestions inline so I am giving it here:

Line 2: This sample can perform Wi-Fi "Scan" operation over both 2.4GHz and 5GHz
Change to : This sample can perform Wi-Fi scan operation in the 2.4GHz and 5GHz bands.

Line 24: Using this sample, the development kit can scan for available Access Points in :abbr:`STA (Station)` mode.
Change to: Using this sample, the development kit can scan for available access points in :abbr:`STA (Station)` mode.

Line 34 and 45: Use code block:: console to write the code.

Line 44: Leave a blank line after line 44.

Line 45: The output should be similar to the following
Change to: The sample shows the following output:

Line 45: Do we have any command which shows the following output, if yes, add it.

Line 47 and 52: Remove  "inf scan:"

**Comment by D-Triveni** [comment]
@richabp Resolved.
> Line 34 and 45: Use code block:: console to write the code.
> 
With present change, we can see code block.

File: samples/wifi/scan/README.rst [hunk 1]
```diff
@@ -7,7 +7,7 @@ Wi-Fi: Scan
    :local:
    :depth: 2
 
-This sample allows the Nordic Wi-Fi chipset to scan for the Access points.
+The Scan sample demonstrates how to use the Nordic Semiconductor's Wi-Fi chipset to scan for the Access points without using the wpa_supplicant.
```
Comment (richabp) [inline samples/wifi/scan/README.rst:10]:
```suggestion
The Scan sample demonstrates how to use the Nordic Semiconductor's Wi-Fi chipset to scan for the access points without using the wpa_supplicant.
```

File: samples/wifi/scan/README.rst [hunk 2]
```diff
@@ -19,9 +19,9 @@ The sample supports the following development kit:
 Overview
 ********
 
-This sample can perform Wi-Fi "Scan" operation over both 2.4GHz and 5GHz
+This sample can perform Wi-Fi scan operation in both 2.4GHz and 5GHz bands.
```
Comment (richabp) [inline samples/wifi/scan/README.rst:22]:
```suggestion
This sample can perform Wi-Fi scan operation in the 2.4GHz and 5GHz bands.
```

File: samples/wifi/scan/README.rst [hunk 3]
```diff
@@ -30,11 +30,6 @@ Building and running
 
 .. include:: /includes/build_and_run_ns.txt
 
-Currently, the following board(s) are supported:
-
-* nRF7002 DK
-
-
 To build for the nRF7002 DK, use the ``nrf7002dk_nrf5340_cpuapp`` build target.
 The following is an example of the CLI command::
```
Comment (richabp) [inline samples/wifi/scan/README.rst:34]:
```suggestion
The following is an example of the CLI command:

.. code-block:: console
```

File: samples/wifi/scan/README.rst [hunk 4]
```diff
@@ -47,11 +42,12 @@ Testing
 
 #. |connect_kit|
 #. |connect_terminal|
-   The output should be similar to the following::
 
-      <inf> scan: Scan requested
+    The sample shows the following output::
```
Comment (richabp) [inline samples/wifi/scan/README.rst:46]:
```suggestion
    The sample shows the following output:
    
    .. code-block:: console
```

**Review by richabp — CHANGES_REQUESTED** [review]

**Review by bama-nordic — APPROVED** [review]
LGTM

**Review by richabp — APPROVED** [review]

**Pull request merged**